components/TutorialStartCard.js

```diff
@@ -0,0 +1,46 @@
+import Image from "next/image";
+import React from "react";
+import styled from "styled-components";
+
+export default function TutorialStartCard() {
+  return (
+    <>
+      <CardContainer>
+        <Image
+          src="/assets/tutorial-start-card_animation_300x300.gif"
+          alt="rocket animation"
+          width={300}
+          height={300}
+        ></Image>
+        <Subline>Let&apos;s get started!</Subline>
+        <p>
+          You&apos;re about to learn something new.
```

Comment:
why you're using `&apos` ?

Reply:
![Bildschirmfoto 2022-10-20 um 09 17 45](https://user-images.githubusercontent.com/71948045/196884060-7fd2a256-3c72-4841-873a-066df4d33e0e.png)
Because eslint suggested to do it this way :D

is there a better way?

Reply:
> ![Bildschirmfoto 2022-10-20 um 09 17 45](https://user-images.githubusercontent.com/71948045/196884060-7fd2a256-3c72-4841-873a-066df4d33e0e.png) Because eslint suggested to do it this way :D
> 
> is there a better way?

oh okey. I don't know 😅